### PR TITLE
model and migration for post_shares

### PR DESCRIPTION
**Pull request by jeromes123**
**OVERVIEW**

model and migration for post_shares

### CODE REVIEW

File: src/app/Models/PostShare.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'post_id',
+        'user_id',
+        'repost_content',
+    ];
+
+    public function post(): BelongsTo
```
Comment (mikco-hipe) [inline src/app/Models/PostShare.php:22]: add doc bloc

File: src/app/Models/PostShare.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        'user_id',
+        'repost_content',
+    ];
+
+    public function post(): BelongsTo 
+    {
+        return $this->belongsTo(UserPost::class, 'post_id');
+    }
+
+    public function user(): BelongsTo
```
Comment (mikco-hipe) [inline src/app/Models/PostShare.php:30]: add doc bloc

File: src/app/Models/PostShare.php (hunk 1 above)
Comment (jeromes123) [reply src/app/Models/PostShare.php:22]: will do, sir

File: src/app/Models/PostShare.php (hunk 2 above)
Comment (jeromes123) [reply src/app/Models/PostShare.php:30]: will do, sir

**Review by mikco-hipe — APPROVED** [review]
LGTM 🙇

**Pull request merged**